docs/ops/activation/LogSoftmax_5.md

```diff
@@ -0,0 +1,42 @@
+## LogSoftMax <a name="LogSoftmax"></a> {#openvino_docs_ops_activation_LogSoftmax_5}
+
+**Versioned name**: *LogSoftmax-5*
+
+**Category**: *Activation*
+
+**Short description**: LogSoftmax computes the log of softmax values for the given input.
```

Comment:
what is a base of logarithm? please give explicit description for it.

Reply:
Base is e, I verified on ONNX. That is usual since log in programming languages denotes natural logarithm not like in math.